containers/Makefile

```diff
@@ -114,22 +114,21 @@ build/concent-api/concent-api.tar: build/repositories/concent/  build/virtualenv
 	mkdir --parents $(dir $@)
 	package-builder/build-concent-api.sh "build/repositories/concent/" "build/concent-api"
 
-
 build/nginx-proxy/concent-api-assets.tar: build/repositories/concent/ build/virtualenv/ build/package-builder/build-concent-api-assets.sh
 	mkdir --parents "$(dir $@)"
-	build/package-builder/build-concent-api-assets.sh \
-		"build/repositories/concent/"                 \
-		"$(abspath package-builder)"                  \
-		"$(abspath build)"                            \
-		"$(dir $@)"
+	build/package-builder/build-concent-api-assets.sh\
+	  "build/repositories/concent/"                  \
+	  "$(abspath package-builder)"                   \
+	  "$(abspath build)"                             \
+	  "$(dir $@)"
 
 build/nginx-proxy/concent-api-docs.tar: build/repositories/concent/ build/package-builder/build-concent-api-docs.sh
 	mkdir --parents $(dir $@)
-	build/package-builder/build-concent-api-docs.sh   \
-		"build/repositories/concent/"                 \
-		"$(abspath package-builder)"                  \
-		"$(abspath build)"                            \
-		"$(dir $@)"
+	build/package-builder/build-concent-api-docs.sh\
+	  "build/repositories/concent/"                \
+	  "$(abspath package-builder)"                 \
+	  "$(abspath build)"                           \
+	  "$(dir $@)"
```

Comment:
You have replaced all indents with 2-space ones. As you can see from the diff, we're already using 4-space indents. Changing it in the middle of an unrelated pull request is a bad idea.

Please keep the original indentation. Just fix the code you have added.

I've updated the issue about coding style to add info about 4-character spaces.